packages/generate-pdf/src/pdf-a-3a/index.ts

```diff
@@ -0,0 +1,42 @@
+import { PDFDocument, PDFName } from 'pdf-lib';
+
+import { addStructTree } from './add-struct-tree';
+import { setColorProfile } from './set-color-profile';
+import { addXMPMetadata } from './xmp-metadata';
+
+export const AUTHOR = 'S1SEVEN';
+export const PRODUCER = 'S1SEVEN Schema Tools';
+export const COLOR_PROFILE_FILE = `${__dirname}/../../../../fixtures/color-profiles/sRGB2014.icc`;
```

Comment:
When using the compiled code, this probably won't resolve to the correct path.
Could you add this file to the generate-pdf package and copy it in the asset during compilation?